db/migrations/state/0004.sql

```diff
@@ -0,0 +1,12 @@
+-- +migrate Up
+CREATE TABLE state.event
```

Comment:
Not sure about storing this table in the state schema. What about create a diferent schema named "motinorization" for instance and store this table there. We can use the same db (stateDB) but without mixing the monitor info with the node state info. WDYT?

Reply:
Yes, I agree. We also have the "degug" table. I think we should open a new issue and to this in the (near) future.

Reply:
It is easier for devops to read a table thant the logs.